packages/flutter/lib/src/material/divider.dart

```diff
@@ -109,6 +109,13 @@ class Divider extends StatelessWidget {
   /// {@end-tool}
   final Color? color;
 
+  /// The border radius of the divider.
+  ///
+  /// This rounds the corners of the divider's line. If null, no rounding
+  /// is applied.
```

Comment:
The default behavior should be in its own newline block.

```suggestion
  /// This rounds the corners of the divider's line. 
  ///
  /// If null, no rounding is applied.
```